projects/ad_gmsl2eth_sl/k26/system_project.tcl

```diff
@@ -23,4 +23,7 @@ adi_project_files ad_gmsl2eth_sl_k26 [list \
   "$ad_hdl_dir/../corundum/fpga/common/syn/vivado/tdma_ber_ch.tcl" \
   "$ad_hdl_dir/library/common/ad_iobuf.v" ]
 
+# Suppress Critical Warning from Corundum source code.
+set_msg_config -id {Designutils 20-1280} -new_severity WARNING
 adi_project_run ad_gmsl2eth_sl_k26
+reset_msg_config -id {Designutils 20-1280} -default_severity
```

Comment:
why are you resetting?

Reply:
Resetting after the synthesis, so it won't affect other projects.

Reply:
On the last commit, I remove the dangling IP instead, not requiring to downgrade this error message